src/components/search-form/search-form.js

```diff
@@ -229,6 +258,77 @@ class SearchForm extends Component {
       );
   }
 
+  renderAccessTypesFilter() {
+    const {
+      accessTypesStoreData,
+      searchByAccessTypesEnabled,
+      searchFilter = {},
+    } = this.props;
+
+    const {
+      'access-type': accessTypes = ''
+    } = searchFilter;
+
+    const {
+      sections,
+    } = this.state;
+
+    const accessTypesList = accessTypes ? accessTypes.split(',').map(accessType => {
+      return accessType.toLowerCase();
+    }) : [];
+
+    accessTypesList.sort();
+
+    const accessStatusTypesExist = !!accessTypesStoreData?.items?.data?.length;
+
+    return accessTypesStoreData?.isLoading
+      ? <Icon icon="spinner-ellipsis" />
+      : accessStatusTypesExist && (
+        <div
+          className={styles['search-filters']}
+          data-test-eholdings-tag-filter
+        >
+          <Accordion
+            label={<FormattedMessage id="ui-eholdings.settings.accessStatusTypes" />}
+            id="accessTypesFilter"
+            separator={false}
+            open={sections.accessTypesFilter}
+            closedByDefault
+            header={FilterAccordionHeader}
+            displayClearButton={accessTypesList.length > 0}
+            onClearFilter={() => this.props.onStandaloneFilterChange({ 'access-type': undefined })}
+            onToggle={this.toggleSection}
+          >
+            {this.renderSearchByAccessTypesCheckbox()}
+            <FormattedMessage id="ui-eholdings.accessTypes.filter">
+              {
+                label => (
+                  <MultiSelectionFilter
+                    id="accessTypeFilter"
+                    ariaLabel={label}
+                    dataOptions={this.getAccessTypesDataOptions()}
+                    name="access-type"
+                    onChange={this.handleStandaloneFilterChange}
+                    selectedValues={accessTypesList}
+                    disabled={!searchByAccessTypesEnabled}
+                  />
+                )
+              }
+            </FormattedMessage>
+          </Accordion>
+        </div>
+      );
+  }
+
+  getAccessTypesDataOptions() {
+    const { accessTypesStoreData } = this.props;
+
+    return accessTypesStoreData.items.data.map(accessType => ({
+      value: accessType.attributes.name.toLowerCase(),
+      label: accessType.attributes.name.toLowerCase(),
```

Comment:
Does value equal to label?

Reply:
yes. back-end filters packages using access status types' labels, not ids